commands/fun/based.js

```
@@ -0,0 +1,182 @@
+    // Create table if it doesn't exist
+    try {
+      await this.database.run(`
+        CREATE TABLE IF NOT EXISTS based (
+          user_id TEXT NOT NULL,
+          guild_id TEXT NOT NULL,
+          based_count INTEGER DEFAULT 0,
+          cringe_count INTEGER DEFAULT 0,
+          PRIMARY KEY (user_id, guild_id)
+        )
+      `);
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used. Allowed unused caught errors must match /^_/u.

(@typescript-eslint/no-unused-vars)